contrib/mypy/examples/src/python/mypy_plugin/BUILD

```diff
@@ -21,6 +21,7 @@ python_library(
   dependencies=[
     ':django-stubs',
   ],
+  tags=["nolint"],
```

Comment:
TIL with the Target API: `tags` can be any iterable. I don't see any reason to use a `set` because it unnecessarily diverges from us using lists everywhere else.

So, I've been using `[]` in docs.

Reply:
Sets vs lists might imply ordering matters? Consistency is good, but not super important.

Reply:
Yeah that's the only thing I was thinking. But that logic also applies to `sources` and `dependencies`, so we should arguably be using sets there too.

Maybe we should use sets in general, but I think lists are fine because they're very recognizable and no shift key :)

Reply:
Yes to sources, no to dependencies... order does matter on the JVM and in Python due to classpath/pythonpath collisions. In dep inference, we'd use import order to play the same role I think.